tests/unit_tests/test_wave_06.py

```diff
@@ -109,7 +121,18 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    assert result 
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
```

Comment:
Great job again! You could test for the items you don't expect to find in each friend's list as well!